Change aux package to util

fickludd opened the pull request:
'aux' is apparently reserved by windows, and project
containing a 'aux' directory cannot be checked out
by windows users!

henriknyman commented on enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_4/codegen/GeneratedMethodStructure.scala:
```diff
@@ -61,7 +62,7 @@ object GeneratedMethodStructure {
 
 
-class GeneratedMethodStructure(val fields: Fields, val generator: CodeBlock, aux: AuxGenerator, tracing: Boolean = true,
+class GeneratedMethodStructure(val fields: Fields, val generator: CodeBlock, util: AuxGenerator, tracing: Boolean = true,
```
This variable name didn't really need to change

fickludd replied:
...so windows doesn't complain about variables named `aux` you think? :)

fickludd committed:
Change cypher 'aux' package to 'util'

'aux' is apparently reserved by windows, and project
containing a 'aux' directory cannot be checked out
by windows users!
